### ISSUE TITLE
0.13.0 on Windows 10: Broken setup icon

### ISSUE THREAD

**Issue by dmxt** [BoostIO/Boostnote#3367]
# Current behavior
It's all good and chill, it only occurs during the install process
![DBrRGJjYuU](https://user-images.githubusercontent.com/3248782/69061258-cd513200-09e6-11ea-9ef7-ad067031f53a.png)

# Steps to reproduce

0. Build a PC
1. Install Windows (I'm sorry) 10 (I'm really sorry)
2. Download setup for Boostnote
3. Click on it to install and observe the icon bug that only occurs during the setup

# Environment

- Version: Unknown; Setup file didn't include the version, it should.
OK, now it's installed and after a long, while looking around for that "About" button because of poor UX design decisions, I finally found it... `Boostnote 0.13.0`

C'mon, guys, I know it should be another issue ticket, but you all know that 'About' is like the last possible button in any app, where it's the most right or bottom. Currently, it's in the middle of the settings 'tab' list. I also had hard time finding settings because I was looking for a cogs icon in the first place.

- OS Version and name: `Windows 10 1903`

---
tl;dr
* Fix that icon that nobody cares about let alone creating an issue post
* Move 'about' to the last thing you can click and/or include version number somewhere visible like the main page of the settings page
* Make settings icon from a strange icon to universal cogs
* Add version to the setup filename so we will have no more (1) (2) in filenames for manual updates, you feel me?

**Comment by Flexo013** [comment]
I think it would be fairly easy to add the version to the BoostnoteSetup.exe like we do for the debian installer: https://github.com/BoostIO/boost-releases/releases/tag/v0.13.0

@Rokt33r is this something you can make sure that happens in future releases?
